Formula/zxing-cpp.rb

```diff
@@ -0,0 +1,38 @@
+class ZxingCpp < Formula
+  desc "C++ port of the ZXing barcode decoder"
+  homepage "https://github.com/zxing/zxing"
+  url "https://github.com/zxing/zxing/releases/download/zxing-2.3.0/ZXing-2.3.0.zip"
+  sha256 "a9b9c7e2cd61c7918dc5f8856c0f91665ab6a109c6cd30ce1795d09bf4baf9c4"
+  head "https://github.com/glassechidna/zxing-cpp.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    if build.stable?
+      cd "cpp"
+    end
+
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+
+      if build.stable?
+        lib.install "libzxing.a"
+        bin.install "zxing"
+
+        cd "../core/src/"
+        Dir.glob "./zxing/**/*.h" do |header|
+          (include/File.dirname(header)).install header
+        end
+      end
+
+      if build.head?
+        system "make", "install"
+      end
+    end
+  end
+
+  test do
+    system bin/"zxing"
```

Comment:
Please include a test that actually does something. Consider installing a test image for that.